src/components/Pages/SignupForm/index.js

```diff
@@ -48,26 +50,47 @@ export const SignupForm = () => {
     e.preventDefault();
     //Validate user inputs
     if (email === "" || password === "") {
-      return setError("Please fill both fields");
+      return;
+      // setError("Please fill both fields");
+    }
+
+    if (
```

Comment:
I think we might be able to improve these if's chain, but we'll get into more detail later